Servers/getting-started-how-to-securely-connect-to-your-server.md

```diff
@@ -0,0 +1,110 @@
+{{{
+  "title": "Getting Started: How to securely connect to your server",
+  "date": "12-10-2015",
+  "author": "Anthony Hakim",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+## You have a shiny new virtual server - now what?
+Congratulations! You've just built a new virtual server on CenturyLink Cloud - we assume you have created a new virtual server.  If not, [follow this other guide first](https://www.ctl.io/knowledge-base/servers/creating-a-new-enterprise-cloud-server/), then return to this guide.
+
+Now what?  How do you securely connect to your server(s)?  Well, just follow the steps below to do so.
+
+## Connect to your shiny new virtual server
+In order to connect to your shiny new virtual server, you will need to:
+
+1. Download and install the OpenVPN client
+2. Download and install the certificate/OpenVPN configuration
+3. Launch the OpenVPN client and connect to your VPN
+4. Launch your Remote Desktop or SSH client, and connect to your server
+
+### 1. Download and install the OpenVPN client
+You will use the OpenVPN client to securely connect to your new virtual server.
```

Comment:
steps 1, 2, 3 are all covered in https://www.ctl.io/knowledge-base/network/how-to-configure-client-vpn/;  this seems like duplicating kb/data for no reason.  if there are updates to using openvpn just update the current guide.  once you get past installing openvpn (which is already documented) there is really 1 step to RDP or SSH.  i'm not sure that warrants a kb.  

openvpn is not the only secure way to connect to servers either.  we have IPSEC Tunnels, CNS, IQ (MPLS) etc.  if this is a kb on the many options to connect those missing seem to limit its usefulness.